### ISSUE TITLE
questions about training NBA dataset

### ISSUE THREAD

**Issue by jiajiaxiaoskx** [fanyun-sun/IMMA#2]
Thanks for providing such a great work！I have some questions when training in the NBA datasets
1. I use the default config in the main.py and run_bball.sh when training the NBA datasets. However, it seems that I have to take weeks to get result for MG+PLT setting and seven or eight days to get result for MG setting. I guess whether the default config is for the other two datasets? Could you please share the config and hyparameters about the NBA datasets?

2. As said above, I use the default config to tain and get ade/fde result as 0.66/1.31, which is quite better than the result in the same setting presented in your paper. So I don't know if there are some errors in the config

3. If the random seeds influence the final result? If it is, could you please share the seeds that lead to the results presented in the paper?

Thanks again for answering me questions listed above!

**Comment by sunfanyunn** [comment]
Indeed, it can take a while to obtain results on the NBA dataset. If you would like to expedite it, consider 
- increasing the initial learning rate (and increasing the rate of decay).
- increase the interval for logging and evaluation

The results in the paper are obtained by averaging the number over multiple runs with different random seeds.

**Comment by jiajiaxiaoskx** [comment]
Thanks for quick reply!
I have one more question in main.py  evaluate function. You write in line 41 that:
        constant = 0.3048 if args.env == 'bball' else 1.
I'm wondering why the NBA datasets have to multiply this constant after scaling. Could you please quote some reference or tell me the logic?

Thanks a lot!

**Comment by sunfanyunn** [comment]
That is to convert the unit from Foot to Meter.

**Comment by jiajiaxiaoskx** [comment]
> Indeed, it can take a while to obtain results on the NBA dataset. If you would like to expedite it, consider
> 
> * increasing the initial learning rate (and increasing the rate of decay).
> * increase the interval for logging and evaluation
> 
> The results in the paper are obtained by averaging the number over multiple runs with different random seeds.

could you please share the ade/fde result when training NBA datasets using all default config in your repo? I get a surprisingly low result.

Also, I would like to ask that what's the FPS of the other two datasets( Phase, Social Navigation Environment) you used in the experiement. 
Thanks a lot!
